Geometry/HGCalCommonData/plugins/DDHGCalWaferFullRotated.cc

```diff
@@ -0,0 +1,239 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
A DD4hep version of the code in this file is required.

Reply:
Please see my comment below